index.js

```diff
@@ -77,6 +74,25 @@ class ClientServiceBase {
     return 'rpc.status-bin';
   }
 
+  /**
+   * get max age
+   * INT32_MAX is equivalent to
+   * java's Integer.MAX_VALUE which is 2147483647
+   * @return {number}
+   */
+  static get maxAge() {
+    const INT32_MAX = 0x7fffffff;
```

Comment:
Defining constant in a method looks weird to me. Is it a common practice to do it in js?

Reply:
Normally in this case, I would just not use a constant, and do it like so
```
static get maxAge() {
    return 0x7fffffff // equivalent to java's Integer.MAX_VALUE which is 2147483647
  }
```

Reply:
That is also fine.